### PR TITLE
Update .NET Core installation instructions for RHEL 7

### PR DESCRIPTION
**Pull request by omajid**
## Summary

Clean up RHEL 7 installation text and make it shorter, more complete and more consistent with rest of the documentation.

## Details

- Fix strange heading and make it more consistent with other headings in this doc
- Remove warnings about removing previous versions - the whole point of SCL is to make everything, including native libraries, installable side-by-side.
- Add separate installation steps for 1.0, 1.1 and 2.0. 

One note: I dont know how to render this doc locally. Some formatting may be off.

### CODE REVIEW

File: docs/core/linux-prerequisites.md [hunk 1]
```diff
@@ -114,11 +110,42 @@ To install .NET Core dependencies on RHEL Server:
...
+
+Install .NET Core 1.0 SDK and Runtime:
+    ```bash
+    yum install rh-dotnetcore10
+    ```
+Enable .NET Core 1.0 SDK and Runtime for your environment:
+    ```bash
+    scl enable rh-dotnetcore10 bash
+    ```
+---
```
Comment (JRAlexander) [inline docs/core/linux-prerequisites.md]: Omair, This is great!
Just one set of changes and we are good to go.
The code snippet sections inside the tab (# [.NET Core 2.x](#tab/netcore2x)) need to be fully left justified to display correctly. Look at the tabs in the Ubuntu section for an example.
Please update those and I will get it merged. Thanks for your contribution!!!

**Review by JRAlexander — CHANGES_REQUESTED** [review]
@omajid  ,
This is great! Just a small set of changes and we will be good to go. Thanks!

**Comment by omajid** [comment]
Thanks for the review. I have updated the spaces in the code segments.

**Review by JRAlexander — APPROVED** [review]

**Comment by omajid** [comment]
Thanks for merging this! Can you let me know when this version goes live?

**Comment by JRAlexander** [comment]
Sorry!

**Pull request merged**